### ISSUE TITLE
[Feature]: Can we add one link to Spark Connector's Documents?

### ISSUE THREAD

**Issue by longdafeng** [oceanbase/spark-connector-oceanbase#1]
### Describe your use case

Right now, There is one page which introduction all third-party's tools for Spark. 

### Describe the solution you'd like

Can we add one link to Spark connector's Documents?

https://spark.apache.org/third-party-projects.html

**Comment by whhe** [comment]
Good idea. We can try it out when the initial implementation is generally ready.

**Comment by whhe** [comment]
It has been updated in https://github.com/apache/spark-website/pull/601
